src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,353 @@
+            "additionalProperties": false,
+            "dependentSchemas": {
+                "average_insert_size": {
+                    "not": {
+                        "required": [
+                            "lower_bound_insert_size",
+                            "upper_bound_insert_size"
+                        ]
+                    },
+                    "comment": "Specification of average_insert_size is mutually exclusive with specification of lower_ and upper_bound_insert_size."
+                },
+                "lower_bound_insert_size": {
+                    "not": {
+                        "required": [
+                            "average_size"
+                        ]
+                    },
+                    "required": [
+                        "upper_bound_insert_size"
+                    ],
+                    "comment": "Specification of lower_bound_insert_size is mutually exclusive with specification of average_insert_size."
+                },
+                "upper_bound_insert_size": {
+                    "not": {
+                        "required": [
+                            "average_insert_size"
+                        ]
+                    },
+                    "required": [
+                        "lower_bound_insert_size"
+                    ],
+                    "comment": "Specification of upper_bound_insert_size is mutually exclusive with specification of average_insert_size."
+                }
+            },
+            "properties": {
+                "average_insert_size": {
+                    "title": "Average Insert Size",
+                    "description": "The average size of the inserts cloned into vectors in the library.",
+                    "type": "number",
+                    "minimum": 0
+                },
+                "lower_bound_insert_size": {
+                    "title": "Lower Bound Insert Size",
+                    "description": "Lower bound of the size of the inserts cloned in vectors in the library.",
+                    "type": "number"
+                },
+                "upper_bound_insert_size": {
+                    "title": "Upper Bound Insert Size",
+                    "description": "Upper bound of the size of the inserts cloned in vectors in the library.",
+                    "type": "number"
+                }
+            }
+        },
+        "guide_library_details": {
+            "title": "Guide Library Details",
+            "description": "Details about a library of guides cloned into constructs.",
+            "type": "object",
+            "additionalProperties": false,
+            "required": [
+                "guide_type"
+            ],
+            "dependentSchemas": {
+                "average_guide_coverage": {
+                    "not": {
+                        "required": [
+                            "upper_bound_coverage_range",
+                            "lower_bound_coverage_range"
```

Comment:
These property names don't match up with the property names below (lines 333-338) e.g. "lower_bound_guide_coverage".

Reply:
Thank you for catching this- too many similarly-named properties. All are adjusted to upper_ and lower_bound_guide_coverage